### ISSUE TITLE
Widespread use of character portrait.

### ISSUE THREAD

**Issue by obiwankennedy** [Rolisteam/rolisteam#52]
This feature request is the next step from #22 

-having the portrait displayed in chat
-the ability to very quickly change between portraits, and/or pile up images to generate a portrait

I will need more details about the pile up images idea. Do you have example ?

**Comment by Blacky---** [comment]
Sure.

Right click on a portrait, if you are GM or if you control that character you get a list of Linked portraits to choose from.

So, Linked Portrait are like a pile. Current portrait at the top, the other behind it. To add Linked portraits, maybe through the current interface for characters?

But, you'll need to think about how do you want to handle thing like “displaying icons to help player notice a state, like paralyzed, turned to stone, hasted, whatever”. Because that Linked Portrait Pile could be used for that, but not exactly the same way.

**Comment by obiwankennedy** [comment]
In rolisteam v1.8 you can set different states and set a picture to each state. This list is control by the game master. But It is close to what you want. If the GM enters each state for each character you will be able to do what right now. Of course, it is a bit painful for GM to define all of that.

The main difficulty will be to offer a proper way to do that.

**Comment by sioc-de-narf** [comment]
Out of standart states.
Like I requested I think in the forum : imagine you have a group of superhero.
Each player want to have her/his character shown on her/his token BUT also to choice between my character is in civil (secret identity) or wearing the masks.
Of course each player wants to have his own images and not sharing a state.
In the same way, have a chapechanger that, we will say have 5 shapes, using only states won't be clear, having 5 images so we know how he looks is.

Is that clearer ?

**Comment by obiwankennedy** [comment]
Yes, I know how to do it. The charactersheet should manage that.

Action in charactersheet will set all shapes for character.
And then it will be easy to switch to another shape.

**Comment by sioc-de-narf** [comment]
:) Thanks
